packages/composer/src/MetadataComposer.ts

```diff
@@ -0,0 +1,60 @@
+import { createClient, Client, OperationResult } from '@urql/core'
+import fetch from 'isomorphic-unfetch'
+
+/**
+ * Merge an array of subgraph query results from different networks
+ * Export this function for testing purposes
+ * @param result array of query results from subgraphs
+ * @returns merged result
+ */
+export function merge(result: OperationResult<any, any>[]): any {
```

Comment:
should merge according to a time stamp? should be an order.

Reply:
the technical issue is that we might not have a timestamp to order by, depending on what fields come back from the result.